spacy/gold.pyx

```diff
@@ -691,35 +761,33 @@ cdef class DocAnnotation:
 
 
 cdef class Example:
-    def __init__(self, doc_annotation=None, token_annotations=None, doc=None,
-                 make_projective=False, ignore_misaligned=False, goldparse=None):
+    def __init__(self, doc_annotation=None, token_annotation=None, doc=None,
+                 goldparse=None):
```

Comment:
Naming here is tricky. These names are a bit long, because we'll want to be calling the constructor with keyword arguments. In fact I'd probably want to make the function keyword only. I also don't know whether it should be `doc_annotation` or `doc_annotations`. `goldparse` should definitely be `gold_parse`.

I'll accept `doc_annotation` and `token_annotation` if we can't think of anything better --- I can't right now. But I do wish we had something shorter.

Reply:
As I said above, I kinda think we could drop both and just have `Example.words` and `Example.cats`, etc. They are a bit too long and it's not quite as useful to have them as when each sentence was a separate unit by default. I think this change should be in a different PR, though, to just focus on the split/merge distinction here.

Reply:
Fair enough, I think that's probably reasonable. But yeah let's get that merged.